src/processors/builtins.py

```diff
@@ -37,3 +37,11 @@ def apply_filter(self, image, _args):
             tuple(self.options.get("kSize", (3, 3))),
             self.options.get("sigmax", 0),
         )
+    
+    
+class MedianBlur(ImagePreprocessor):
+    
+    def apply_filter(self, image, _args):
+        return cv2.medianBlur(
+                            image,
+                            self.options.get("kSize", 5))
```

Comment:
We are trying to remove the hardcodings. So for any new code to be introduced, take it via options in the constructor.
```
def __init__(self, options, _args):
     self.kSize = options.get("kSize", 5)
```
Then use it in apply_filter.